10-JavaScript/classes.md

```diff
@@ -0,0 +1,289 @@
+# JavaScript Classes
```

Comment:
This resource is great and really thorough. We need to add more interaction and exercises at each stage/new topic because as-is, there will be too much content with no practical application. 

My suggestions are:
- Put prototype introduction in a separate resource
- After the constructor and instance methods sections, add an exercise to write a class that does something
- Enhance that class with getter/setters (specific instructions for how we'd want to include this)
- Enhance that class with static methods (specific instructions for how we'd want to include this)

Reply:
Ok I've moved the prototypical object to a separate file, pulling from our old one, and added a set of exercises having them build a similar `Animal` class.

Reply:
The exercises look good. They're well dispersed throughout the resource.